### ISSUE TITLE
useless due censorship

### ISSUE THREAD

**Issue by cerulliber** [black-forest-labs/flux#25]
you reach chinese kling censorship level. this is good model but useless due censorship. if it hasn't breasts, I don't wanna see it.

**Comment by shivshankar11** [comment]
But he has valid point about censorship, i think flux said their model has accurate anatomy which is not true, shat is the point of this kind of censorship? are we adults or child who need need protection from human anatomy.

**Comment by HenryThe1st** [comment]
agree... this is not close to accurate anatomy... Its a great model but if there will be no possibility to improve in some aspects with LORAS it will be dead at arrival...

**Comment by shivshankar11** [comment]
without finetune and lora support, community engagement will be significantly less.
but i am hopeful dev version will support finetuning. maybe pro version itself is finetuned with better database.

**Comment by HenryThe1st** [comment]
> What are you guys talking about? The model is uncensored, but wasn't trained on certain images like porn if that's what you were after. That doesn't mean it's censored. Hell, I just tested it, it creates accurate boobies. It's all good.

It actually cannot make accurate breasts (no real nipples). Nevertheless, the real downfall will happen if no LORA are permitted

**Comment by cerulliber** [comment]
> 
> Yes it can. I just tested again: realistic nipples, nice looking boobs.
> 
> Prompting it correctly seems like the issue around here.
> 
> Note that I am using the Dev version, maybe you guys are using Schnell, which may have this issue?

can you proof it ? for demonstrative purposes only.

**Comment by cerulliber** [comment]
> cerulliber, Revln9, and zewt reacted with thumbs down emoji
> 
> 

are you so censorship-oreinted that you even can't support one emoji in online space?

**Comment by shivshankar11** [comment]
well accurate anatomy mean accurate genitalia, flux fail terribly here due to blackmask pasted on dataset. we call that censorship right.

**Comment by ElSchulzoML** [comment]
I mean - breasts are o.k'ish, but vagaga is protected.

**Comment by BrechtCorbeel** [comment]
Although I do not look into using this for NSFW content I do agree that where Midjourney falls apart is that sometimes prompting and veering slightly NSFW enhances a look and feel or a level of sophistication you cannot reach without it, but on the other hand I am 100% agreeing on the danger of a model that can produce nudity and images of kids.

I think people overestimate the danger of "dangerous" text or images, we've been doing this for years now and really I have not yet ever really seen extreme adverse effects, it's anticipated by many and people a desensitized to it.

**Comment by eddiepot** [comment]
It _is_ censored! They purposely cover up the beautiful female body with pasties! Now _that_ is sick!

**Comment by apollo0510** [comment]
This thing is completely censored. It refuses to create anything that is slightly NSFW. At least not by default. Even explicit prompts create child safe images.
